drafts/WEP0002-slack-community-moderation-procedures.rst

```diff
@@ -0,0 +1,146 @@
+
+At the end of the day, we're all human and we all make mistakes that we then learn from. The goal here isn't perfect behaviour, but to ensure that we set an example for the rest of the community in terms of treating each other well, and being able to disagree with each other respectfully.
+
+Post removal
+------------
+When a user posts obvious spam or posts with objectionable content, any one moderator is free to remove the material upon seeing it. Before removing the message, they should:
+
+1. Share the message with the #slack-mods channel to notify the rest of the moderators of what happened, and to keep a record of the deleted message.
+2. Send a note to the poster referencing why the material was removed, and offer more information (including a link to the Code of Conduct) if necessary.
+
+When a moderator removes a post, they may also decide to remove other users' responses. Sometimes this decision is appropriate given the context, and sometimes the discussion that follows (often involving community self-moderation) is worth keeping so members' responses can be visible.
+
+There's a certain amount of personal discretion that goes into removing posts. **When in doubt, the moderation team should generally refrain from removing posts.** A lack of discretion in removing posts can negatively impact the community's trust in the moderation team.
+
+Sales-y content
+---------------
+Historically, the moderation team has struggled with the definition of "sales-y content" listed on the Slack guidelines. We currently define sales-y content as follows:
+
+* Direct pitches
+* Links without context
+* Brands
+
+Some further proposed definitions for the purpose of moderating are as follows:
+
+* **Unpublished market research** - being a helpful bunch, the Write the Docs community is never hesitant to weigh in on their experiences. However, market research that the community contributes to should also benefit the community. Therefore, we ask that by posting a survey and asking for responses, a user commits to allowing the community to access the results for free, and we reserve the right to decline or remove a survey if we don't think that it'll benefit the community sufficiently. We could also create a #surveys channel so users who aren't interested in surveys don't see them. (Academic surveys for a user's studies don't have to be held to the same standards, but a channel for surveys could also theoretically benefit users who are in school and are looking for information about the field.)
```

Comment:
In general, I'm not quite sure what's going on with this point. The heading is about market research, but the text is all about surveys. As written, I worry that a) there's a non-survey research loophole, b) it's unclear about unpublished research or research published in subscription journals, c) and it leaves out unaffiliated researchers.

Perhaps we could reframe it to be less narrowly focused on surveys and to avoid the two-pronged academic/non-academic distinction?

Something like:

> Surveys, solicitations for interview subjects, and other research may be conducted via the Write the Docs Slack if it is accompanied by a commitment that the results of that research will contribute to the broader documentarian community. For example, a survey conducted with a pledge to make the results freely available or to use the data in a scholarly paper would satisfy this requirement.

That's not a complete rewrite—I really like and think we should retain the "we reserve the right to decline or remove [research efforts] if we don't think that it'll benefit the community sufficiently" text—but that's what I'm getting at here: we want to be open to stuff that advances the field as a whole (even though it might not directly benefit WTD specifically).

Reply:
Haha, yeah, this was in response to what I mostly remembered as a bunch of surveys that people were using for market research. I really like the idea of broadening the rule beyond surveys, and stole your wording :)